operator/deploy/olm-catalog/0.2.0/admin.clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: automation-broker-operator-admin
+roleRef:
+  name: admin
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: automation-broker-operator
+  namespace: placeholder
```

Comment:
personally, I think you should make this `openshift-ansible-service-broker` so that we can demonstrate this working.

Reply:
yeah, we obviously may have to do that, hence the WIP tag. I'm just really hoping I screwed something up and this is the correct thing to do.